app/src/minApi21/java/mod/hey/studios/build/BuildSettingsDialogBridge.java

```diff
@@ -11,8 +12,13 @@
 public class BuildSettingsDialogBridge {
     public static String[] getAvailableJavaVersions() {
         return new String[] {
-                SETTING_JAVA_VERSION_1_7,
-                SETTING_JAVA_VERSION_1_8,
+                SETTING_JAVA_VERSION_1_7
```

Comment:
Please at least make a separate commit for this change.

Reply:
you couldn't compile java 8 in minApi21 anyways